styles/Layout/PanelSideBarLayout.scss

```diff
@@ -246,7 +246,7 @@ section.content:first-of-type {
   }
 }
 
-@include media-breakpoint-up(md) {
+@include media-breakpoint-up(xs) {
```

Comment:
was this changed on purpose?

Reply:
Yes, because if the window size is lower than `md` then is not possible to collapse/expand the sidebar panel

Reply:
sounds good :)